src/components/Badge.tsx

```diff
@@ -21,16 +22,67 @@ const StyledBadge = styled(Tag)`
     }
 `;
 
-type BadgeProps = {
+const StyledWrapperDiv = styled.div`
+    + .usa-tooltip__body {
+        font-family: "Source Sans Pro Web";
+    }
+`;
+
+type ConditionalBadgeProps =
+    | {
+        showTooltip?: false;
+        tooltipText?: never;
+    }
+    | {
+        showTooltip?: true;
+        tooltipText: string;
+    };
+
+type CommonBadgeProps = {
     bgColor: 'blue' | 'yellow';
     Icon: React.FunctionComponent<React.SVGProps<SVGSVGElement>>;
     text: string;
 };
 
-export default function Badge({ Icon, bgColor, text }: BadgeProps) {
-    return (
-        <StyledBadge background={`var(--badge-bg-${bgColor})`}>
-            <span className="badge-icon"><Icon /></span>{' '}<span className="badge-text">{text}</span>
+type BadgeProps = CommonBadgeProps & ConditionalBadgeProps;
+
+// ForwardRef of wrapper div around Tag component
+type WrapperDivProps = React.PropsWithChildren<{
+    className?: string;
+}> &
+    JSX.IntrinsicElements['div'] &
+    React.RefAttributes<HTMLDivElement>;
+const WrapperDivForwardRef: React.ForwardRefRenderFunction<
```

Comment:
If we don't need the custom font family, then do we need the forward ref ?

Reply:
The forward ref is needed to attach the tooltip to the div. According to the [docs](https://trussworks.github.io/react-uswds/?path=/story/components-tooltip--custom-component), using a ForwardRef is how to use a custom element instead of the default button looking thing.